addons/README.md

```diff
@@ -221,8 +221,9 @@ When the config resource is pre-created, addons-controller will use it in lieu o
 #### Pre-create ClusterBootstrap resource
 
 If the use case is to swap out a specific package, for example CNI with your own package then create the ClusterBootstrap object in its entirety
-with the same name as the Cluster resource. Creating this resouce in its entirety can put additional burden on clients so a defaulting webhook is
-provided.
+with the same name as the Cluster resource. In case where a custom ClusterBootstrap object was provided, the ClusterBootstrap object will not be
+cloned from the ClusterBootstrapTemplate anymore. Creating the ClusterBootstrap resource in its entirety can put additional burden on clients so a
+defaulting webhook is provided.
 
```

Comment:
I think we also should call out the annotation on the cluster object as a separate mechanism

Reply:
Sure, made this change.